### PR TITLE
Update to Swift 5

### LINKED ISSUE #7: Update for Swift 5 and newer

### PR DESCRIPTION
**Pull request by Gray-Wind**
Should resolve #7

### CODE REVIEW

**Comment by mkj-is** [comment]
Thank you for your contribution!

Unfortunately, we need to make some more changes to make it work. Continuous integration build failed.

- Fix failing test due to wrong Swift version for test target.
- Update tools versions using `bundle update`.
- Update podspec.
- Update README (and version if you want).

**Review by mkj-is — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Unfortunately, we need to make some more changes to make it work. Continuous integration build failed.

- Fix failing test due to wrong Swift version for test target.
- Update tools versions using bundle update.
- Update podspec.
- Update README (and version if you want).

File: Example/Sonar.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -512,6 +514,7 @@
 			baseConfigurationReference = 4F8E94279BF27B78A880A6AE /* Pods-Sonar_Tests.debug.xcconfig */;
 			buildSettings = {
 				BUNDLE_LOADER = "$(TEST_HOST)";
+				DEVELOPMENT_TEAM = S2P73LZ2ZH;
```
Comment (mkj-is) [inline Example/Sonar.xcodeproj/project.pbxproj]: I guess this small change should not have been commited.

```suggestion
```

**Review by mkj-is — CHANGES_REQUESTED** [review]

File: Example/Sonar.xcodeproj/project.pbxproj (hunk 1 above)
Comment (Gray-Wind) [reply Example/Sonar.xcodeproj/project.pbxproj]: Oh, sorry!

**Review by mkj-is — APPROVED** [review]
Thank you very much!

**Pull request merged**